CONTRIBUTING.md

```diff
@@ -2,8 +2,9 @@
 
 ## Bug Reports
 
-Please [open an issue](https://github.com/serokell/ariadne/issues) if
-you find a bug.
+Please [open an issue](https://issues.serokell.io/newIssue?project=AD)
```

Comment:
This page results in 403. I think it would be better to link to https://issues.serokell.io/issues/AD instead.